source/lexbor/encoding/encoding.c

```diff
@@ -218,3 +218,35 @@ lxb_encoding_decode_function_noi(lxb_encoding_t encoding)
 {
     return lxb_encoding_decode_function(encoding);
 }
+
+lxb_status_t
+lxb_encoding_data_call_encode_noi(lxb_encoding_data_t *encoding_data, lxb_encoding_encode_t *ctx, const lxb_codepoint_t **cp,
```

Comment:
@kostya 

limit 80 column. This is not a strict rule. In this case, it is difficult to meet this limit, but you can slightly reduce the line. You should get something like:
```C
lxb_encoding_data_call_encode_noi(lxb_encoding_data_t *encoding_data, lxb_encoding_encode_t *ctx,
                                  const lxb_codepoint_t **cp, const lxb_codepoint_t *end)
```